### ISSUE TITLE
[FEATURE] LLVM IR transpiling support

### ISSUE THREAD

**Issue by Cr0a3** [Cr0a3/ygen#39]
**Is your feature request related to a problem? No

**Describe the solution you'd like**
I would love if you could activate llvm if parsing with a flag.
So then we could run ygen on the LLVM tests

**Comment by Cr0a3** [comment]
TODO-LIST:

 - Implement llvm parser
 - Implement ir node translator to ygen

**Comment by yingmanwumen** [comment]
So as far the ir of ygen is different from llvm ir?

**Comment by Cr0a3** [comment]
> So as far the ir of ygen is different from llvm ir?

Yes a little simpler. It doesn't have as many ir nodes. For example:
- there is no fadd the ygen add node works with floats and ints 
- many of the casting nodes are combined into an actual casting node (sext, text) are not available just cast
